evc-sumo/docker/token.txt

```diff
@@ -0,0 +1 @@
+<your-github-token>
```

Comment:
Yeah, this is precisely what we'd want to avoid. If we're using a file to store the token, we'd want to explicitly add this token.txt to our repos .gitignore so that it isn't ever accidentally committed to the repo. We don't want people accidentally publishing their tokens for our private repos.

Reply:
Hi Kyle, so in this case would it be better for me to just ask the user for a input of the token?

Reply:
removed in the latest version